### PR TITLE
Add `NumberMinMaxFilterType` code exemple

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add `NumberMinMaxFilterType` code exemple
| Fixed ticket? | N/A

### CODE REVIEW

File: src/content/1.7/development/components/form/types-reference/number-min-max-filter.md [hunk 1]
```diff
@@ -18,8 +18,36 @@ The `NumberMinMaxFilterType` represents two `NumberType` fields - one holds mini
...
+
+This type is built for grid filters usage but can be used in forms as well.
+
+Add `NumberMinMaxFilterType` to your form.
+
+```php
+<?php
+use Symfony\Component\Form\AbstractType;
+use PrestaShopBundle\Form\Admin\Type\NumberMinMaxFilterType;
+class SomeType extends AbstractType
```
Comment (Progi1984) [inline src/content/1.7/development/components/form/types-reference/number-min-max-filter.md:31]:
```suggestion
use PrestaShopBundle\Form\Admin\Type\NumberMinMaxFilterType;

class SomeType extends AbstractType
```

File: src/content/1.7/development/components/form/types-reference/number-min-max-filter.md [hunk 2]
```diff
@@ -18,8 +18,36 @@ The `NumberMinMaxFilterType` represents two `NumberType` fields - one holds mini
...
 
+## Code example
+
+This type is built for grid filters usage but can be used in forms as well.
+
+Add `NumberMinMaxFilterType` to your form.
+
+```php
+<?php
+use Symfony\Component\Form\AbstractType;
```
Comment (Progi1984) [inline src/content/1.7/development/components/form/types-reference/number-min-max-filter.md:29]:
```suggestion
<?php

use Symfony\Component\Form\AbstractType;
```

**Review by Progi1984 — COMMENTED** [review]
Some spaces

**Comment by PululuK** [comment]
Done !
Thanks @Progi1984

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @PululuK

**Pull request merged**